recipes/catlearn/meta.yaml

```
@@ -0,0 +1,73 @@
+{% set name = "catlearn" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/C/CatLearn/CatLearn-{{ version }}.tar.gz
+  sha256: c71f6395e984fec3d081ac8fb169188126ac4bd7b3e61620df6272cf0b18af04
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - ase>=3.17.0
```

There needs to be a space between the package name and the version constraint
```suggestion
    - ase >=3.17.0
```
(and for the others)

---

@chrisburr I added the corresponding spaces.